client/src/routes/recover.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'preact/hooks'
+import withStyles from '../components/jss'
+import EnvelopeOpen from '../icons/envelope-open.svg'
+import Form from '../components/form'
+import { recover } from '../api/auth'
+
+export default withStyles({
```

Comment:
Exporting like this produces a component with no display name; prefer declaring the component as a variable and exporting `withStyles({...}, Recover)`